js/w3c/templates/cgbg-headers.html

```diff
@@ -1,6 +1,6 @@
 <div class='head'>
   <p>
-    <a href='http://www.w3.org/'><img width='72' height='48' src='https://www.w3.org/Icons/w3c_home' alt='W3C'/></a>
+    <a href='http://www.w3.org/'><img width='72' height='48' src='https://www.w3.org/Icons/w3c_home' alt='W3C'></a>
```

Comment:
As an aside, we need to push the w3c to start allowing srcset here... the logo now looks blurry and terrible on retina displays.